pages/visualizers/memory-palace/memory-palace.css

```
@@ -0,0 +1,449 @@
+.mp-cell:hover { border-color: #10b981; transform: scale(1.15); z-index: 2; }
+
+.mp-cell.mp-free    { background: rgba(255,255,255,0.03); border-color: rgba(255,255,255,0.06); }
+.mp-cell.mp-data    { background: rgba(16,185,129,0.35);  border-color: #10b981; color: #bbf7d0; }
+.mp-cell.mp-pointer { background: rgba(6,182,212,0.3);    border-color: #06b6d4; color: #a5f3fc; }
+.mp-cell.mp-header  { background: rgba(168,85,247,0.3);   border-color: #a855f7; color: #e9d5ff; }
+.mp-cell.mp-sp      { background: rgba(245,158,11,0.35);  border-color: #f59e0b; color: #fde68a; }
+.mp-cell.mp-active  { background: rgba(34,197,94,0.5);    border-color: #22c55e; color: #fff; animation: mpFlash 0.4s ease; }
+.mp-cell.mp-freed   { background: rgba(239,68,68,0.15);   border-color: rgba(239,68,68,0.3); color: #fca5a5; }
+.mp-cell.mp-selected{ box-shadow: 0 0 0 2px #f59e0b; z-index: 3; }
+
+@keyframes mpFlash {
```

Stylelint (17.13.0)

[error] Expected keyframe name "mpFlash" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)